.github/workflows/build.yaml

```
@@ -610,60 +670,41 @@ jobs:
-  #           macOS-releases/*
-
-  #     - name: post release URL
-  #       run: |
-  #         echo "::notice::Release ${{ steps.release.outputs.url }}"
+      # forked from softprops/action-gh-release
+      - name: Create GitHub release
+        id: release
+        uses: AlchemyViewer/action-gh-release@v3
+        with:
+          # name the release page for the branch. We want channel and version.
+          name: "${{ needs.build.outputs.viewer_channel }} ${{ needs.build.outputs.viewer_version }}"
```

actionlint (1.7.12)

[error] property "viewer_channel" is not defined in object type {imagename: string; velopack_exclude: string; velopack_icon: string; velopack_installer_base: string; velopack_mac_bundle_id: string; velopack_mac_main_exe: string; velopack_mac_pack_id: string; velopack_mac_pack_title: string; velopack_mac_pack_version: string; velopack_main_exe: string; velopack_pack_id: string; velopack_pack_title: string; velopack_pack_version: string; viewer_version: string}

(expression)